### PR TITLE
Improve the github issue template

### PR DESCRIPTION
**Pull request by medyagh**

### CODE REVIEW

**Comment by medyagh** [comment]
I signed it

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+-->
+
+
+**The operating system version**:
+
+**The exact command to reproduce it**:
+
+**The full output of the command that failed**:
+
+**The minikube logs (use `minikube logs` command)**:
```
Comment (tstromberg) [inline .github/ISSUE_TEMPLATE]: Is it possible shorten this statement without reducing clarity? Perhaps just: "The output of the `minikube logs` command

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -1,5 +1,14 @@
-<!-- Thank you for sharing your experience! If you are reporting a bug, please include: -->
-<!--    *  The exact command-lines used so that we can replicate the issue -->
-<!--    *  The full output of the command that failed -->
-<!--    *  The output of the "minikube logs" command, if applicable -->
-<!--    *  Which operating system version was used -->
+<!-- Please use this template while reporting a bug and provide as much info as possible specially minikube logs. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
```
Comment (tstromberg) [inline .github/ISSUE_TEMPLATE]: Remove "specially minikube logs". Chances are, the issue reporter does not know what these are or how to get them.

**Comment by medyagh** [comment]
I signed it

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
+
+-->
+
+
+**The operating system version**:
+
+**The exact command to reproduce it**:
```
Comment (tstromberg) [inline .github/ISSUE_TEMPLATE]: Be more specific than "it", such as "the issue", "the failure", or "the bug". 

Tip: https://kubernetes.io/docs/contribute/style/style-guide/ has some good advice here along these lines.

**Comment by medyagh** [comment]
i signed it

**Pull request merged**